src/include/osl_pvt.h

```diff
@@ -1028,6 +1039,18 @@ class Opcode {
     /// different form.  Only opname is changed.
     void transmute_opname(ustring opname) { m_op = opname; }
 
+    /// Op would require masking under batched execution
+    /// when its arguments are not uniform (varying)
+    bool requires_masking() const { return m_requires_masking; }
```

Comment:
Looking at the names they might be too close to each other, despite proper meanings:
```
bool requires_masking() const
void require_masking()
```
Maybe this would be better
```
bool requires_masking() const
void make_masking_required()
```

Reply:
The idiom I tend to use is:

    bool requires_masking() const;   // return the current value
    void requires_masking(bool val);   // set new value

Reply:
I started with that and realized it was a one way ticket, only was being set to true with default of 0.  So changed the API to reflect that.  But its just data, and idiom used elsewhere so can change it back to that.